uber/models.py

```diff
@@ -2063,8 +2079,18 @@ def is_expired(cls):
         return cls.expiration_date < localized_now()
 
     @hybrid_property
+    def is_free(self):
+        return not self.discount or (
+            self.discount_type == self.PERCENT_DISCOUNT and self.discount > 99)
```

Comment:
Haha, I literally did it that way because it's fewer characters. It is less clear, though. And clarity is the most important thing, so I'll make that change.